TappNetwork/home-visiting-app-web
# Cu 3azfwtg UI feedback

# Details
Changes to UI based on feedback from demo. New Cards styling to replace Ion-Cards. More incorporation of branding elements. Lighter background. Narrower width of directory elements in web mode. Overhauled visitor, supervisor, program, and external resource profile pages. 

# Steps to test
1. Navigate around in both web and mobile modes and make sure everything looks fine.

# Task Linked
[3azfwtg](https://app.clickup.com/t/3azfwtg)

# Visual Changes
![Screen Shot 2022-08-18 at 10 00 53 AM](https://user-images.githubusercontent.com/29612767/185445994-001e8f9f-de07-4a96-9899-b3a362cac7c1.png)
![Screen Shot 2022-08-18 at 10 02 01 AM](https://user-images.githubusercontent.com/29612767/185446030-b66465ab-895a-4fa7-b1f9-384004d48369.png)
